asyncapi/github/service_types.bal

```diff
@@ -49,23 +50,25 @@ public type PullRequestService service object {
     remote function onEdited(PullRequestEvent payload) returns error?;
 };
 
-# Actions related to pull request reviews. The action can be one of submitted, edited or dismissed.
+# Triggers when Choreo recieved a a new event from GitHub related to a GitHub pull request review.
```

Comment:
multiple `a`

Reply:
`Triggers when Choreo recieved a a new event from GitHub related to a GitHub pull request review.` idea is not clear for me 
`GitHub related to a GitHub`

Reply:
Do we need to say `GitHub` explicitly in this?

Reply:
Thanks a lot for the suggestion. Resolved